eb

```diff
@@ -124,11 +126,21 @@ for argument in "$@"; do
 				letter=${letter,} # lowercase first letter
 				name=${argument%%-*}
 				# some names contain - (e.g. HOOMD-blue) so we need pathname expansion
-				ec=$(echo "$EASYBUILD_ROOT/easyconfigs/$letter/$name"*"/$argument")
-				cd $(dirname $ec)
+				ec=$(find $EASYBUILD_ROOT/easyconfigs/$letter/$name* -name $argument 2>/dev/null)
+				if [[ -f $ec ]]; then
+					NEW_ARGS+=("$ec")
+				else    # if no corresponding easyconfig was found in easyconfigs folder, we get it found normally
```

Comment:
I think we should print an error here instead, to avoid looking in ebfiles_repo.

Reply:
🤔 that's something I routinely do however. All of the `--try-*` recipes end up with the modified name in the `eb_repo` directory, and it's something much easier to take one of those than to figure out what's the original recipe and reuse it.  And combining `--try-*` options does not always work in one go, so the original recipe might not even work. 

No longer supporting that may be a bit of a hassle.

Reply:
Fixed.

Reply:
You were right about ebfiles_repo actually. But may still be something to avoid for the reasons I said in the meeting.
(The present `eb` without this patch simply cd's to the directory but uses the original argument, thereby prioritizing the
current directory over ebfiles_repo... I just tested it myself with `sudo -iu ebuser eb OSU-Micro-Benchmarks-5.6.2-iompi-2020a.eb` which succeeds even though it's only in ebfiles_repo.

So please revert your last patch, sorry!

Reply:
😆 done